### DISCUSSION TITLE
Can someone provide a video tutorial on how to use this repo?

### DISCUSSION THREAD

**Discussion by Njasa2k** [cloneofsimo/lora#102] [General]
I feel I would be able to use LORA if someone was able to film a tutorial video on how to use it.

**Comment by bmaltais** [comment]
You can always give a try to this: https://github.com/bmaltais/kohya_ss

There is a LoRA GUI that make it pretty easy to create and use the created network.

Follow the quickstart video on how to do a dreambooth... https://github.com/bmaltais/kohya_ss/blob/master/README_dreambooth.md

Then simply use the new "Dreambooth LoRA" tab to generate a lora instead:

![image](https://user-images.githubusercontent.com/7474674/210265726-037b5ab8-277b-4090-9de3-6e5ff0fbb0e8.png)

The LoRA specific settings are highlighted in red:

![image](https://user-images.githubusercontent.com/7474674/210265872-e5c7867d-f324-44a5-930f-1aa0774314c0.png)

Then install the auto1111 extension to load the resulting LoRA network:

https://github.com/kohya-ss/sd-webui-additional-networks

It should look like this in auto1111:

![image](https://user-images.githubusercontent.com/7474674/210266045-668a3c9d-668b-4a63-9e90-a6eb32db1ead.png)

**Reply by ExponentialML** [reply]
Awesome! I didn't know you have implemented this. Thank you.

**Comment by brian6091** [comment]
@FurkanGozukara posted a video:
https://github.com/cloneofsimo/lora/issues/105#issuecomment-1368562865

**Reply by FurkanGozukara** [reply]
> @FurkanGozukara posted a video: [#105 (comment)](https://github.com/cloneofsimo/lora/issues/105#issuecomment-1368562865)

thank you so much for referring me

**Comment by cloneofsimo** [comment]
Nerdy Rodent made a decent tutorial as well : https://www.youtube.com/watch?v=gw2XQ8HKTAI
But do note that these tutorials are all outdated : they are using v0.0.2 of this LoRA implementation

**Reply by FurkanGozukara** [reply]
> Nerdy Rodent made a decent tutorial as well : https://www.youtube.com/watch?v=gw2XQ8HKTAI But do note that these tutorials are all outdated : they are using v0.0.2 of this LoRA implementation

I did 2 days. Where can I see the Lora version? and how can I use the latest version? I think that may be the cause why LoRA is pretty inferior
